### ISSUE TITLE
How to use the SID?

### ISSUE THREAD

**Issue by Mrils** [hufu6371/DORN#11]
Hello.

Thanks for your awesome paper and I'd prepared to reproduce your training code.

But I have a question that how to use the SID? It seems like we need to calculate K thresholds by SID in the data loading stage and transmit the thresholds to loss layers. Is right?

Thanks.

**Comment by hufu6371** [comment]
@Mrils Yes, the K thresholds are predefined in our paper. Pls. see Eq. 1.

**Comment by junweifu** [comment]
I have a problem about the value of alpha and beta in SID. When I use NYU-v2 to train our model, how to determine those two values. Are these two values represent the absolute depth?

**Comment by pobingwanghai** [comment]
@junweifu I guess you should calculate the min and max depth value among the whole dataset for NYU and KITTI. Then you get the alpha and beta value and you can define the K bu yourself but the author proved 80 is the best in sec.4.2.3.
